### ISSUE TITLE
Training code

### ISSUE THREAD

**Issue by LucasSLD** [zhangy76/PhysPT#16]
Hello,

I was looking through the code and I could not find the training code (loss computation and training loop of PhysPT model). Is it available somewhere?

Best regards

**Comment by zhangy76** [comment]
Thanks for reaching. The training part is not available. However, please feel fee to reach for any specific issue you may encouter when implementing the method.
